Libplanet/Store/Trie/IKeyValueStore.cs

```diff
@@ -13,6 +13,8 @@ public interface IKeyValueStore : IDisposable
 
         public void Set(byte[] key, byte[] value);
 
+        public void Set(IDictionary<byte[], byte[]> values);
```

Comment:
Although there are other methods lacking docs too, could you write a short doc XML for this method (and `<inheritdoc cref="IKeyValueStore.Set(IDictionary{byte[], byte[]})"/>` to the implementations)?  It would be good to write docs for new added code.

Reply:
Amended. https://github.com/planetarium/libplanet/pull/1570/commits/7a047b6c6ed40432caefb3018e4b2cc2e1a2ec9f